release-notes/kvm/v11.1.0.md

```diff
@@ -3,3 +3,8 @@
 ### net-exporter [v1.6.0](https://github.com/giantswarm/net-exporter/blob/master/CHANGELOG.md#160-2020-01-29)
 
 - Allow disabling TCP DNS check.
+
+### nginx-ingress-controller v0.28.0 ([Giant Swarm App v1.3.0](https://github.com/giantswarm/nginx-ingress-controller-app/blob/master/CHANGELOG.md#v130-2020-01-30))
```

Comment:
```suggestion
### nginx-ingress-controller v0.27.1 ([Giant Swarm App v1.2.0](https://github.com/giantswarm/nginx-ingress-controller-app/blob/master/CHANGELOG.md#v120-2020-01-21))
```

@sslavic does that look ok? (the old URL was broken)

Reply:
@kopiczko yes it does